server/system/service/user.go

```diff
@@ -1139,3 +1150,23 @@ func (svc user) DeleteAvatar(ctx context.Context, userID uint64) (err error) {
 
 	return svc.recordAction(ctx, uaProps, UserActionDeleteAvatar, err)
 }
+
+func processAvatarInitials(u *types.User) (initials string) {
```

Comment:
You've mentioned you'd use the email also; that no longer the case?

Reply:
That is no longer the case. When a user is created, their handle is generated, and since a handle has a higher priority in this case, I opted to remove the user's email from the ` processAvatarInitials` function.